tests/test_convert_lit_checkpoint.py

```diff
@@ -191,3 +191,79 @@ def test_maybe_unwrap_state_dict(tmp_path):
     with mock.patch("scripts.convert_lit_checkpoint.maybe_unwrap_state_dict") as maybe_unwrap:
         convert_lit_checkpoint(checkpoint_name=ckpt_name, out_dir=tmp_path, model_name=model_name)
     maybe_unwrap.assert_called()
+
+
+def test_maybe_raise_finetune_warning_adapter(tmp_path):
```

Comment:
You'll need to update these test names too

Reply:
implemented in [3638666](https://github.com/Lightning-AI/lit-gpt/pull/397/commits/3638666d91b1ee6d00306212f09a9d37a658cdc6)